webapp/channels/src/components/suggestion/suggestion_list.tsx

```diff
@@ -272,14 +219,10 @@ export default class SuggestionList extends React.PureComponent<Props> {
                 continue;
             }
 
-            if (isSelection) {
-                this.currentItem = item;
-            }
-
             items.push(
                 <Component
                     key={term}
-                    ref={(ref: any) => this.itemRefs.set(term, ref)}
+                    id={`suggestionList_item_${term}`}
```

Comment:
Ideally, we'd probably have a unique ID for each instance of `SuggestionList`, but the `ul` element for this below always has the ID `suggestionList` that's referred to by a ton of E2E tests, so I don't want to change that now